src/js/components/editor/Editor.vue

```diff
@@ -83,14 +162,14 @@ export default {
   mounted() {
     // check to see if there's a columnID in the route
     // if so, attempt to load up the text straightaway
-    const colID = this.$route.params.colID
-    if (colID !== '~' && colID > 0) {
-      this.getText(this.$route.params.scrollVersionID, colID)
-    }
+    this.refresh()
```

Comment:
This should not be necessary.  In AppBody.vue at the website startup I save any initial route, toggle it to ~ for everything and then back to that saved initial route so that the regular watchers in each component will pick up on any initial route.  This saves writing and thinking about `mounted` routines like this in the components; initial loading of the site triggers a watchable change.